packages/playwright-test/src/fixtures.ts

```diff
@@ -131,12 +147,13 @@ export class FixturePool {
       for (const entry of Object.entries(fixtures)) {
         const name = entry[0];
         let value = entry[1];
-        let options: Required<FixtureOptions> | undefined;
+        let options: { auto: boolean, scope: FixtureScope, option: boolean, timeout: number | undefined } | undefined;
```

Comment:
what was the reason that this got changed? I thought TypeScript-wise this is the same.

Reply:
Nope, it insists on `timeout` being a number, not undefined.